### PR TITLE
Remove the msg dependency from style

### PR DESCRIPTION
**Pull request by bholley**
This is part of slimming down the dependencies of geckolib.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9261)

### CODE REVIEW

File: python/tidy.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
     os.path.join(".", "target", "*"),
     os.path.join(".", "ports", "gonk", "src", "native_window_glue.cpp"),
     os.path.join(".", "ports", "cef", "*"),
+    os.path.join(".", "ports", "geckolib", "*"),
```
Comment (jdm) [inline python/tidy.py]: I don't think we need to ignore this, do we?

Comment (bholley) [inline python/tidy.py]: Good catch - that was an accidental leftover from rebasing this over what actually landed in #9209.

**Comment by KiChjang** [comment]
This is puzzling... #9254 didn't even land yet!

**Comment by bholley** [comment]
fixes #9246

**Pull request merged**